src/current/v23.2/set-up-physical-cluster-replication.md

```diff
@@ -137,20 +142,50 @@ The standby cluster requires a user profile on the primary cluster to connect to
 
 ### Connect to the primary virtual cluster (optional)
 
-1. If you would like to connect to the primary's application virtual cluster, open a new terminal window, and use the `ccluster=application` parameter:
+1. If you would like to run a sample workload on the primary's application virtual cluster, open a new terminal window and use [`cockroach workload`]({% link {{ page.version.version }}/cockroach-workload.md %}) to run the workload.
+
+    For example, to initiate the [`movr`]({% link {{ page.version.version }}/movr.md %}) workload:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    cockroach workload init movr "postgresql://root@{node IP or hostname}:{26257}/?options=-ccluster=application&sslmode=verify-full&sslrootcert=certs/ca.crt&sslcert=certs/client.root.crt&sslkey=certs/client.root.key"
```

Comment:
I don't think `cockroach workload` accepts a connection string, although it doesnt accept `--secure`. Can you verify? When I tried to pass it a connection string, it did not work. If you are running on the default SQL port, `cockroach workload` seems to hit that port and do the right thing. Otherwise I think you can give it a port directly. https://www.cockroachlabs.com/docs/stable/cockroach-workload

Reply:
Ah ok, so I was working directly on the cockroach SQL port, so it worked just fine with the connection string. 

There is the `port=` param which seems to work. How did you get your `cockroach workload` to run? I don't know which flag would work to pass in a diff flag?

Reply:
I just ran it locally on one of the nodes (the one that was listening on the default port, as it happens). I did see that you have added src/current/_includes/v23.2/connect/cockroach-workload-parameters.md and I didn't try doing it using a postgresql:// address or didn't know that was a possibility. So maybe this is fine how you have it, and in fact better than mine, which was working in spite of me rather than because of me.